03/Task03.js

```diff
@@ -1,25 +1,30 @@
 import React from 'react';
-
-import { Row, Col, Breadcrumb as RBBreadcrumb } from 'react-bootstrap';
+import Breadcrumb from "../src/components/Breadcrumb";
+import breadcrumbTheme from "../src/components/Breadcrumb/theme";
+import { ThemeProvider } from "styled-components";
+import { Row, Col, Breadcrumb as RBBreadcrumb } from "react-bootstrap";
 
 const Task03 = () => {
-    return (
-        <Row>
-            <Col>
-                <RBBreadcrumb>
-                    <RBBreadcrumb.Item href="#">Home</RBBreadcrumb.Item>
-                    <RBBreadcrumb.Item href="https://getbootstrap.com/docs/4.0/components/breadcrumb/">
-                        Library
-                    </RBBreadcrumb.Item>
-                    <RBBreadcrumb.Item active>Data</RBBreadcrumb.Item>
-                </RBBreadcrumb>
-            </Col>
-            <Col>
-                Breadcrumb!
-            </Col>
-        </Row>
-)
-}
+  console.log(breadcrumbTheme);
+  return (
+    <Row>
+      <Col>
+        <RBBreadcrumb>
+          <RBBreadcrumb.Item href="#">Home</RBBreadcrumb.Item>
+          <RBBreadcrumb.Item href="https://getbootstrap.com/docs/4.0/components/breadcrumb/">
+            Library
+          </RBBreadcrumb.Item>
+          <RBBreadcrumb.Item active>Data</RBBreadcrumb.Item>
+        </RBBreadcrumb>
+      </Col>
+      <Col>
+        <ThemeProvider theme={breadcrumbTheme}>
+          <Breadcrumb></Breadcrumb>
```

Comment:
Struktura powinna być podobna do tej, którą widzimy powyżej - wtedy jest to bardziej elastyczne :)